### DISCUSSION TITLE
Expedition 2 Atlas

### DISCUSSION THREAD

**Discussion by GMANMOTG** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#22] [General]
I was able to finish all of this expedition except phase 4 the Atlas mission.  No matter what I do I cannot find an Atlas and polo doesn't seem to want to help.  I saw a couple youtubes but appears 5 rendezvous has been changed so another system so youtube is irrelevant.  If I am missing something please let me know!

**Comment by cwmonkey** [comment]
@GMANMOTG Ask him to tell you where black holes are and it'll show you black holes and atlas stations on your map. I believe this is in the readme's. Re-open if that doesn't work for you.

**Reply by GMANMOTG** [reply]
You mean black holes in stead of atlas?  I asked him for atlas and he shows nothing

**Reply by GMANMOTG** [reply]
I also jumped for about 3 hours hoping to just run into one and it doesn't seem they are spawning while offline and in expedition mode.

**Reply by cwmonkey** [reply]
> You mean black holes in stead of atlas?

@GMANMOTG Yes, ask for black holes, and he will show you Atlas AND Black Holes. It's a bug in their code.
